examples/llm/components/prefill_worker.py

```diff
@@ -115,6 +121,7 @@ async def generate(self, request: RemotePrefillRequest):
             is_remote_decode=True,
             decode_block_ids=request.block_ids,
             decode_engine_id=request.engine_id,
+            decode_computed_block_ids=request.computed_block_ids,
```

Comment:
I am still going through the changes in the vLLM patch for the remote block ids, but I am confused as I don't see any changes in the Vllmworker side to push this blocks to prefillworker. 
I would assume the request to the prefillWorker  from the vllmworker should populate computed_block_ids.